nanome/beta/nanome_sdk/session/session_client.py

```diff
@@ -96,10 +94,7 @@ async def update_structures_deep(self, structures):
         message_type = Messages.structures_deep_update
         expects_response = True
         args = structures
-        request_id = self._send_message(message_type, args, expects_response)
-        result = await self.request_futs[request_id]
-        del self.request_futs[request_id]
-        return result
+        self._send_message(message_type, args, expects_response)
 
     def update_structures_shallow(self, structures):
```

Comment:
shouldn't this and `update_structures_deep` still return a future so they can be awaited? since `expects_response = True`

Reply:
fixed update_structures_deep, but update_structures_shallow doesn't expect a response according to the plugin_instance implementation, so the fix was to set `expects_response=False`